Array & Hash/KadaneAlgo.cpp

```diff
@@ -0,0 +1,55 @@
+#include <vector>
+#include <algorithm>
+#include <limits>
+using namespace std;
+
```

Comment:
Problem link & Time complexities of Solutions are missing

Reply:
Oh yes, I'll add them.